app/tests/src/test_ingest_ca_public_charge.py

```diff
@@ -9,16 +9,39 @@
 from src.db.models.document import Document
 from src.ingest_ca_public_charge import _ingest_ca_public_charge
 
-from .test_ingest_edd_web import check_database_contents, sample_cards  # noqa: F401
+from .test_ingest_edd_web import check_database_contents  # noqa: F401
 
 
 @pytest.fixture
-def sample_markdown(sample_cards):  # noqa: F811
-    items = json.loads(sample_cards)
-    for item in items:
-        item["h4"] = item["title"]
-        item["markdown"] = item.get("main_content", item.get("main_primary"))
-    return json.dumps(items)
+def sample_markdown():  # noqa: F811
+    return json.dumps(
+        [
+            {
+                "url": "https://keepyourbenefits.org/en/ca/",
+                "title": "CALIFORNIA - Public Charge, Benefits and Immigration",
+                "markdown": "\n### Who is affected by the Public Charge Rule?\n\n* It does not apply to:\n- U.S. Citizens or people applying for citizenship.  \n  - Lawful Permanent residents (Green Card holders) unless the Green Card holder leaves the U.S. for more than 6 months. A Public Charge assessment can apply when they try to return.",
+                "main_content": "\n### Who is affected by the Public Charge Rule?\n\n* It does not apply to:\n- U.S. Citizens or people applying for citizenship.  \n  - Lawful Permanent residents (Green Card holders) unless the Green Card holder leaves the U.S. for more than 6 months. A Public Charge assessment can apply when they try to return.",
```

Comment:
If `markdown` and `main_content` are the same, don't populate `main_content` and just use `markdown` instead in the spider.